index.html

```diff
@@ -247,10 +412,12 @@ <h2>I'm always interested in hearing about new projects, so if you'd like to cha
                         <label for="comment"></label>
                         <textarea name="message" maxlength="500" required id="comment"  cols="30" rows="5" placeholder="Write me something..."></textarea>
                     </div>
-                    <button class="btn-form" type="submit">Get in touch</button>
+                    <button class="btn-form btn-form-desktop" type="submit">Get in touch</button>
                 </form>
             </section>
+            <!-- FORM SECTION END HERE -->
 
+            <!-- FOOTER SECTION START HERE -->
             <footer class="footer-page">
```

Comment:
- [ ] As per the previous reviewer's feedback, it would be great to check the **top and bottom spacing of the icon's container inside your `footer`.** There seems to be too much space between the icons and the top and bottom edges of the container, which may cause your project to deviate from the desired design. 💯

  _HINT:
  To address this, you may want to consider decreasing the values for the top and bottom padding or margin in the container, which will help create a more balanced and aesthetically pleasing layout._ 

  ### Your design:
  ![image](https://user-images.githubusercontent.com/107952724/236361104-7ea4d55d-ef15-4756-9a08-cd69aa0a0d05.png)
  
  ### Figma design:
  ![image](https://user-images.githubusercontent.com/107952724/236361181-de2fbb9e-ee93-4c36-94e3-b161266d2316.png)